microsoft-edge/devtools-guide-chromium/accessibility/test-analyze-no-focus-indicator.md

```diff
@@ -1,51 +1,76 @@
 ---
-title: Analyze the lack of indication of keyboard focus in a sidebar menu
-description: Analyzing the lack of indication of keyboard focus in a sidebar menu, due to lacking a CSS pseudo-class rule for the focus state on a link, combined with the link having no outline setting.
+title: Analyze the lack of indication of keyboard focus
+description: Analyzing the lack of indication of keyboard focus, due to missing CSS pseudo-class rules for the focus state on a link, combined with the link having no outline setting.
```

Comment:
```suggestion
description: Analyzing the lack of indication of keyboard focus, due to missing a CSS pseudo-class rule for the focus state on a link, combined with the link having no outline setting.
```
often singular is clearer (eg prevents many-to-one ambiguities).  in this case, d/k.  tech review this suggested change.  @captainbrosset

Reply:
Agreed. Adding this change.